### PR TITLE
WinUI-3213 : Updated mask types in SfMaskedTextBox UG document.

### PR DESCRIPTION
**Pull request by Vignesh-SF3580**
### Description
Updated the mask types page for SfMaskedTextBox control.

### Solution Description
Added the mask types page and added content about mask types.
Added the output images for the mask type.

### CODE REVIEW

File: winui/Masked-TextBox/Mask-Types.md [hunk 1]
```diff
@@ -7,4 +7,192 @@ control: SfMaskedTextBox
...
+{% highlight C# %}
+
+SfMaskedTextBox maskedTextBox = new SfMaskedTextBox();
+maskedTextBox.MaskType = MaskedTextBoxMaskType.RegEx;
+maskedTextBox.Mask = "(0[0-9]|1[1-9]|2[1-3]){2}:[0-5][0-9]:[0-5][0-9]";
+
+{% endhighlight %}
+{% endtabs %}
+
+![Mask type RegEx in WinUI MaskedTextBox](MaskedTextBox_Images/winui_mask_type_regex.png)
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Mask-Types.md:196]: @Vignesh-SF3580 Change the code snippet with the setting Value property and add a single image for both simple and regex.

Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Mask-Types.md]: Added gif images for both simple and regex mask.

File: winui/Masked-TextBox/Mask-Types.md [hunk 2]
```diff
@@ -7,4 +7,192 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Mask types in WinUI SfMaskedTextBox
```
Comment (Manivannan-E) [inline winui/Masked-TextBox/Mask-Types.md:10]: @Vignesh-SF3580 Change the description.

Learn more about the various mask types available in the MaskedTextBox (SfMaskedTextBox) control, including Simple and RegEx, and how to use them effectively.

Comment (Vignesh-SF3580) [reply winui/Masked-TextBox/Mask-Types.md:10]: Changed the description.

**Review by Manivannan-E — APPROVED** [review]
Changes are fine.

**Pull request merged**